scripts/run_mypy_checks.py

```diff
@@ -47,6 +47,7 @@
     'core/controllers/android_e2e_config_test.py',
     'core/controllers/base.py',
     'core/controllers/base_test.py',
+    'core/controllers/beam_jobs.py',
```

Comment:
We are not expecting to add new files in this list, I think we have added types in this file and can remove this line from here, right?

Reply:
Done.